helm/cosmo/charts/router/templates/_helpers.tpl

```diff
@@ -43,6 +43,15 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Additional Labels that are just rendered in metadata.labels
+*/}}
+{{- define "router.additionalLabels" -}}
```

Comment:
I tested it with multiple labels and I got an error. Seems like for some reason the whitespace trimmer `-` caused it.

Can you check again.

```yaml
additionalLabels: 
  fantasy.unicorn.io/name: rainbow-router
  fantasy.unicorn.io/instance: sparkle-instance
  fantasy.unicorn.io/version: 0.197.1
  fantasy.unicorn.io/managed-by: MagicWand
  fantasy.unicorn.io/component: fairy-dust-router
```

```suggestion
{{- define "router.additionalLabels" }}
```